sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/view/ViewsImpl.scala

```diff
@@ -93,6 +101,9 @@ final class ViewsImpl(system: ActorSystem, _services: Map[String, ViewService])
               val commandName = receiveEvent.commandName
               val msg = service.messageCodec.decodeMessage(receiveEvent.payload.get)
               val metadata = MetadataImpl.of(receiveEvent.metadata.map(_.entries.toVector).getOrElse(Nil))
+              if (TraceId.isValid(metadata.traceContext.traceId())) {
```

Comment:
Still feels a bit weird to not have it `None` and not do anything on missing rather than return invalid and then validate that to know that we don't have one, seems cheap enough though I guess.

Reply:
So you think is better` Optional<String> traceId();` in `TraceContext`. And not follow their style, but more and Scala-ish.

Reply:
done